.circleci/config.yml

```diff
@@ -84,7 +84,7 @@ jobs:
             echo $GCLOUD_SERVICE_KEY > ~/gcloud-service-key.json
             gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
             export UNISWAP_CONTRACT_DATA="uniswap-contract-data.txt"
-            export BONDED_ECDSA_KEEP_FACTORY_CONTRACT_DATA="BondedECDSAKeepFactory.json"
+            export BONDED_ECDSA_KEEP_VENDOR_CONTRACT_DATA="BondedECDSAKeepVendor.json"
```

Comment:
In `circleci-provision-external-contracts.sh` we still talk about `BONDED_ECDSA_KEEP_FACTORY_ADDRESS` and set `BondedECDSAKeepFactoryAddress`